### PR TITLE
samtools markdup: add stats output

### PR DESCRIPTION
**Pull request by bernt-matthias**
fixes https://github.com/galaxyproject/tools-iuc/issues/2557

also fixes indentation mayhem 

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tool_collections/samtools/samtools_markdup/samtools_markdup.xml [hunk 1]
```diff
@@ -30,45 +30,57 @@ $stats
...
+    2> '$stats_output'
+#end if
     ]]></command>
     <inputs>
         <param name="bamfile" type="data" format="sam,bam,cram" optional="false" label="Alignment" />
         <param name="remove" type="boolean" argument="-r" truevalue="-r" falsevalue="" label="Remove duplicate reads" />
         <param name="maxlen" type="integer" optional="true" argument="-l" min="0" label="Expected maximum read length of INT bases. (default 300)"/>
-	<param name="stats" type="boolean" argument="-s" truevalue="-s" falsevalue="" label="Print basic statistics" />
-	<param name="supp" type="boolean" argument="-S" truevalue="-S" falsevalue="" label="Mark supplementary reads of duplicates as duplicates" />
+        <param name="stats" type="boolean" argument="-s" truevalue="-s" falsevalue="" label="Print basic statistics" />
```
Comment (bgruening) [inline tool_collections/samtools/samtools_markdup/samtools_markdup.xml]: Something for the new QC extension :)

https://github.com/galaxyproject/galaxy/pull/7913

Comment (bernt-matthias) [reply tool_collections/samtools/samtools_markdup/samtools_markdup.xml]: could you explain a bit?

Comment (bgruening) [reply tool_collections/samtools/samtools_markdup/samtools_markdup.xml]: Sorry to confuse you, this was just a note, nothing urgent. The new 19.09 version of Galaxy will have (thanks to @jmchilton) a new feature that will let us extract some quality matrices from std* and annotate results with this.

See here a small example: https://github.com/galaxyproject/galaxy/pull/7913/files#diff-fcb2a01adf509f1b963b67f76a070590R3

IUC should discuss what we can do with that and implement this for a few tools imho.

Comment (jmchilton) [reply tool_collections/samtools/samtools_markdup/samtools_markdup.xml]: ❤️

**Pull request merged**